CHANGELOG.md

```diff
@@ -34,6 +34,7 @@ To learn more about Ansible Builder and Execution Environments, see: https://www
 - Added user interface for the activity stream: https://github.com/ansible/awx/pull/9083
 - Converted many of the top-level list views (Jobs, Teams, Hosts, Inventories, Projects, and more) to a new, permanent table component for substantially increased responsiveness, usability, maintainability, and other 'ility's: https://github.com/ansible/awx/pull/8970, https://github.com/ansible/awx/pull/9182 and many others!
 - Added click-to-expand details for job tables
+- Added search filtering to job output https://github.com/ansible/awx/pull/9208
```

Comment:
(Duplicate)
```suggestion
```

Reply:
@jakemcdermott it ended up getting removed in https://github.com/ansible/awx/commit/f6fb3e0b41da23d50aacdf710bc6353f2a1da1ea because we didn't think this PR was going to land in time. This just reverts that commit.